### ISSUE TITLE
Transfer to DAppchain never succeeds

### ISSUE THREAD

**Issue by KojimaGeniy** [loomnetwork/cards-gateway-example#4]
I'm doing everything right by instruction, `./transfer_gateway setup`, `./transfer_gateway start`, import 0xbb63b692f9d8f21f0b978b596dc2b8611899f053d68aec6c1c20d1df4f5b6ee2, sign in the message with this account, send transaction with token transfer, everything goes without problems but no tokens appear under DAppChain Account tab. 
I tried upgrading loom from 371 to 375, tried checking if ganache and loom are working, and they are reachable, tried changing ganache to truffle develop.
Also I had problem as described [here](https://github.com/loomnetwork/cards-gateway-example/issues/1), solved by manually migrating or pasting address instead of `DC_CRYPTO_CARDS_JSON.networks[networkId].address,` or by `./transfer_gateway restart`

**Comment by eduardonunesp** [comment]
There are some inconsistencies we're going to fix them for sure, are you able to run the example?

**Comment by KojimaGeniy** [comment]
@eduardonunesp Yes, eventually it worked, but I noticed that somehow it doesn't work with loom 375, and only with 371. 
And I still don't know why it doesn't work when I'm trying to replace `ganache-cli `with `truffle develop`. I'm doing migrations, changing ports and gateway address in `loom.yaml` and port in mapping script, then running everything else. Nothing else changes, but still, I see no tokens coming to DAppChain, and because of that I can't see how to use it in Testnet or Mainnet. Is it even possible?

**Comment by eduardonunesp** [comment]
Thanks @KojimaGeniy we're going to update the example and it will be possible to use testnet or mainnet also

**Comment by peara** [comment]
I have found out that after click `Send to DAppChain`, the card owner is Gateway address and Gateway contract knew about the deposit.
But there is no log in `oracle.log`, so I guess the issue is from GatewayOracle.
I have check the mapping and it seems to work fine.
I can locate the ERC721Received event as well.

ETH transfer and ERC20 transfer work well and there are logs in `oracle.log`.
